backend/index.js

```diff
@@ -42,16 +58,38 @@ app.get("/api/posts/:limit", (req, res) => {
   );
 });
 
+app.post("/api/postimg", upload.single('file'), (req, res) => {
```

Comment:
Why do you do this ? Shouldn't it be part of the POST /user, given the name of the PR ?